unitTestSupportFunctions/getBodyNodesOnRectangularDomain.m

```diff
@@ -0,0 +1,81 @@
+function homDOFs = getBodyNodesOnRectangularDomain (fldMsh, homDBC)
```

Comment:
This function can be removed from the repository as it serves no more any purpose after you modified the input file and the parser since the underlying information can now be retrieved nicely from the GUI.